platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -350,6 +350,12 @@ sep24:
     account_creation: false
     claimable_balances: false
 
+  # When the `POST /transactions/deposit/interactive` and `POST /transactions/withdraw/interactive` are called,
+  # the request may contain KYC fields. If `kyc_fields_forwarding` is enabled, the SEP server will call the
+  # `PUT /customer` endpoint of the business server to create or update the KYC fields.
+  kyc_fields_forwarding:
+    enabled: true
+
```

Comment:
I think it would be preferable if this config object wasn't necessary. Instead, the platform can check if SEP-12 is enabled when KYC fields are passed by wallets, and if so, it can make the `PUT /customer` request. What do you think?

Reply:
I think your proposed would be smoother. 

To be cautious, is there a use case when SEP-12 is enabled but SEP-24 does not want to forward KYC fields?

Reply:
> What if the anchor supports sep-31? How are they going to differentiate between sep-31 and sep-24 customers? By type? And do they need to do so?

I don't think our callback API differentiate them though.

Reply:
The Platform would need to use built-in types for SEP-24 requests since the wallet isn't providing it directly. We can use something like `sep24-deposit-customer`.

> To be cautious, is there a use case when SEP-12 is enabled but SEP-24 does not want to forward KYC fields?

I don't think so. I can't envision a case where the business wouldn't want to offer that functionality. If they don't support it but still have SEP-12 enabled that is fine too. They just won't prefill the UI and the user will have to re-enter the KYC.

Reply:
Hey @JakeUrban , I think it is possible that the anchor does not want to open SEP-12. But they want SEP-24 to receive the KYC via `kycFieldsForwarding`

Reply:
Ok, then we can keep this config and SEP-12 doesn't have to be enabled.